src/libcmis/ws-relatedmultipart.cxx

```diff
@@ -240,7 +240,7 @@ string RelatedMultipart::addPart( RelatedPartPtr part )
     return cid;
 }
 
-void RelatedMultipart::setStart( string& cid, string& startInfo )
+void RelatedMultipart::setStart( string& cid, const string& startInfo )
```

Comment:
Why haven't you changed `cid` as well?

Reply:
Because it failed to build with this.

Reply:
Ok you were right, I hadn't changed RelatedMultipart::getPart method.
I've just pushed another commit on the PR.